### PR TITLE
Replace start using a service screenshots

### PR DESCRIPTION
**Pull request by owenatgov**
## Fix
This is in response to an issue raised by DAC that the screenshots we use on the [Start using a service pattern](https://design-system.service.gov.uk/patterns/start-using-a-service/) are putting content after the start button which is an accessibility issue.

Fixes https://github.com/alphagov/govuk-design-system/issues/4026 (more details in issue)

## Notes
There's a question on if we should also remove or amend the line just below the second screenshot which references the Carer's Allowance service since we're no longer using that service as a reference.

### CODE REVIEW

**Comment by selfthinker** [comment]
I think the screenshots work well as replacements.
But I agree that the text needs to be changed as well.

(It's interesting that the text first refers to "applying for probate" although it's not related to either of the original screenshots. I think it does make sense out of context without any of the images, it's just a bit weird.)

**Comment by owenatgov** [comment]
I've tried to adjust the following sentence to still link to the blog but de-emphasise the mention of the Carer's Allowance guide. Probably one for @calvin-lau-sig7 to review when he has time.

File: src/patterns/start-using-a-service/index.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ It’s important that users have this context, so it makes sense to embed the st
 
 ![A screenshot showing an example of a start point within a multipart guide. The page is shown as part of a set of pages and its body text explains what the user will need to use the service, followed by a start button labelled 'Apply now'.](multipart-guide-start-point.png)
 
-The Carer’s Allowance team found that, for their service, [providing a start point within a multipart guide helped users get to the right information without feeling overwhelmed](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
+You can read more about the benefits of providing a start page within a multipart guide in [this blog post by DWP's digital team talking about how they improved the Carer's Allowance guide](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
```
Comment (calvin-lau-sig7) [inline src/patterns/start-using-a-service/index.md:70]:
```suggestion
[Read a blog post about how DWP's digital team found that providing a start page within a multipart guide for Carer's Allowance improved the journey for users](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
```

**Review by calvin-lau-sig7 — COMMENTED** [review]
I've made a suggestion to try to give the de-emphasising hist we're going for. To be honest, I don't think the edit is absolutely needed at all but it doesn't hurt.

**Comment by owenatgov** [comment]
@calvin-lau-sig7 I like it! If I do some commit squashing are you happy to approve?

**Review by calvin-lau-sig7 — APPROVED** [review]

**Pull request merged**